### ISSUE TITLE
No export conditions for svelte field in package.json

### ISSUE THREAD

**Issue by alstat** [umaranis/svelte-lexical#74]
Hi, I tried using this library with the following steps:

```
npm create svelte@latest my-app
cd my-app
npm install
npm install svelte-lexical
npm run dev -- --open
```
And then, I get the following warning.
```
3:26:14 PM [vite-plugin-svelte] WARNING: The following packages have a svelte field in their package.json but no exports condition for svelte.

svelte-lexical@0.3.5

Please see https://github.com/sveltejs/vite-plugin-svelte/blob/main/docs/faq.md#missing-exports-condition for details.
```
Opening this issue because the Vite documentation tells us to inform the library maintainer when we see this. No errors though with the built app so far, just that warning.

**Comment by umaranis** [comment]
Hi @alstat ,

I have added the export condition with [cc1efc4](https://github.com/umaranis/svelte-lexical/commit/cc1efc40c50db936730d2218ca2c6917447a8bdf)

**Comment by umaranis** [comment]
resolved with version 0.4.0
